internal/integration/rest/utils_test.go

```diff
@@ -7,42 +7,88 @@ import (
 )
 
 func TestExtendProjects(t *testing.T) {
```

Comment:
ExtendProjects doesn't seem to exist anymore.

Reply:
I'll fix this :)